.changeset/seven-books-dig.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-confidential-contracts': minor
+---
+
+Add `ERC7984Freezable` extension.
```

Comment:
`ERC7984Freezable`: Add an extension to `ERC7984`, which allows accounts granted the "freezer" role to freeze and unfreeze tokens.